index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>freeCodeCamp Survey Form</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div class="container">
+    <div class="header">
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">"Thank you for taking time to help us improve the platform."</p>
```

Comment:
Please try to write it in a more descriptive way. So, that user could understand more better  😊👌👍😎